src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
Would something like this work then? https://github.com/enumag/symfony/commit/cb59e7cefa4fcfc8a011161a9acb781a59e04ffe